vllm/spec_decode/spec_decode_worker.py

```diff
@@ -409,6 +409,14 @@ def execute_model(
             execute_model_req)
         num_lookahead_slots = execute_model_req.num_lookahead_slots
 
+        all_prompt = (all(
```

Comment:
I am wondering if we can add an e2e test for the case with speculative-draft-tensor-parallel-size > 1 and chunked-prefill enabled.

Reply:
Let me take a look and see

Reply:
Added, has an issue with multiple prompts in batch. Investigating.